src/aggregate/functions/date.c

```diff
@@ -60,19 +60,20 @@ static int timeFormat(ExprEval *ctx, RSValue *result, RSValue **argv, size_t arg
 
 /* Fast alternative to mktime which is dog slow. From:
 https://gmbabar.wordpress.com/2010/12/01/mktime-slow-use-custom-function/ */
```

Comment:
I am concerned that this blog post will die eventually. should we replace with an archive link?

Reply:
I'm not sure there is one. And even if so, this blog-post implementation is not very good and our implementation is more of the idea of implementing it ourselves instead of using `mktime`